### DISCUSSION TITLE
Image File problem

### DISCUSSION THREAD

**Discussion by stevepurton** [kubilus1/autoortho#188] [Q&A]
Installed North America 3 times but keep getting following error when I try to run X Plane...............
Error could not locate image file for terrain
Custom Scenery/z_na06/terrain/25856_19024_B116_sea_overlay.ter
Custom Scenery/z_na_06/terrain/
../textures/25856_19024_B116.dds

Any suggestions how I fix this please
Thanks

**Comment by kubilus1** [comment]
Make sure that autoortho is running, as in after install you hit the 'run' button.  It looks like possibly it's not running.  If it did run, are there errors in the logs for autoortho (Users dir .autorotho-data/logs)

**Comment by stevepurton** [comment]
Yes thanks, problem solved
